src/legacy/core_plugins/kibana/public/context/app.html

```diff
@@ -62,48 +62,20 @@
   ng-if="contextApp.state.loadingStatus.anchor.status !== contextApp.constants.LOADING_STATUS.FAILED"
 >
   <!-- Controls -->
-  <div class="kuiBar kuiVerticalRhythm">
-    <div class="kuiBarSection">
-      <context-loading-button
-        data-test-subj="predecessorLoadMoreButton"
-        is-disabled="![
-          contextApp.constants.LOADING_STATUS.LOADED,
-          contextApp.constants.LOADING_STATUS.FAILED,
-        ].includes(contextApp.state.loadingStatus.predecessors.status)"
-        icon="'fa-chevron-up'"
-        ng-click="contextApp.actions.fetchMorePredecessorRows()"
-      >
-        <span
-          i18n-id="kbn.context.loadMoreDescription"
-          i18n-default-message="Load {defaultStepSize} more"
-          i18n-values="{
-            defaultStepSize: contextApp.state.queryParameters.defaultStepSize
-          }"
-        ></span>
-      </context-loading-button>
-      <context-size-picker
-        count="contextApp.state.queryParameters.predecessorCount"
-        data-test-subj="predecessorCountPicker"
+  <context-action-bar
```

Comment:
Not super important and I might be wrong about that, but isn't `predecessors` and `successors` used the wrong way? A predecessor is `a thing that has been followed or replaced by another.`, so this should be the older documents, right? But here it is fetching the newer ones.

Reply:
exactly, here predecessor are the docs that are displayed before the anchor, but I also do agree. I'd go with e.g. `newer`, `older`, I think then there are no more questions open. However, there's much more to refactor in this case, functions etc. 
that's why I kept this heritage for now (but I'd like to change it in a future refactoring)